docs/book/src/developer/providers/implementers-guide/building_running_and_testing.md

```diff
@@ -52,17 +53,29 @@ Conditions:
 ```
 
 [capi]: https://github.com/kubernetes-sigs/cluster-api
+[releases]: https://github.com/kubernetes-sigs/cluster-api/releases
 
 ### Your provider
 
+In this guide, we are building an _infrastructure provider_. We must tell cluster-api and its developer tooling which type of provider it is. Edit `config/default/kustomization.yaml` and add the following common label. The prefix `infrastructure-` [is used](label_prefix) to detect the provider type.
```

Comment:
```suggestion
In this guide, we are building an _infrastructure provider_. We must tell cluster-api and its developer tooling which type of provider it is. Edit `config/default/kustomization.yaml` and add the following common label. The prefix `infrastructure-` [is used][label_prefix] to detect the provider type.
```
to fix the link / the link checker

You can directly quash after this fix if you want

Reply:
Okay, that's fixed now. And why would we squash commits? This is a collection of various, unrelated changes where each commit message clearly describes the rationale. Therefore it's better to merge all of them instead of squashing.

Reply:
It's all now one commit

Reply:
Thx!